### PR TITLE
New package: SudachiEmu.Sudachi version 1.0.2

### PR DESCRIPTION
**Pull request by xiSage**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/159882)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\SudachiEmu.Sudachi__DefaultSource\QtWebEngineProcess.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\SudachiEmu.Sudachi__DefaultSource\sudachi-cmd.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\SudachiEmu.Sudachi__DefaultSource\sudachi.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\sudachi.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 886.)

**Comment by stephengillie** [comment]
HI @xiSage,

Does this package have any dependencies? (The message mentions `Microsoft.VCRedist.2015+.x64` (or .(`x86`) because they are the vast majority of dependencies.)

**Comment by xiSage** [comment]
> HI @xiSage,
> 
> Does this package have any dependencies? (The message mentions `Microsoft.VCRedist.2015+.x64` (or .(`x86`) because they are the vast majority of dependencies.)



> HI @xiSage,
> 
> Does this package have any dependencies? (The message mentions `Microsoft.VCRedist.2015+.x64` (or .(`x86`) because they are the vast majority of dependencies.)

I think this package should have only one dependency on Microsoft Visual C++ Redistributable.

File: manifests/s/SudachiEmu/Sudachi/1.0.2/SudachiEmu.Sudachi.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: sudachi.exe
+  PortableCommandAlias: sudachi
+UpgradeBehavior: install
+Commands:
+- sudachi
+ReleaseDate: 2024-05-27
+Installers:
```
Comment (stephengillie) [inline manifests/s/SudachiEmu/Sudachi/1.0.2/SudachiEmu.Sudachi.installer.yaml:15]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\SudachiEmu.Sudachi__DefaultSource\QtWebEngineProcess.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\SudachiEmu.Sudachi__DefaultSource\sudachi-cmd.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\SudachiEmu.Sudachi__DefaultSource\sudachi.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\sudachi.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 886.)

**Comment by stephengillie** [comment]
> I think this package should have only one dependency on Microsoft Visual C++ Redistributable.

I agree - the issue is that this manifest currently has zero dependencies, when that one is needed.

**Comment by stephengillie** [comment]
Unfortunately, the symlink doesn't work.
image

Verify from full link:
image

**Pull request closed without merging**